tests/features/collection/api.feature

```diff
@@ -17,3 +18,31 @@ Feature: Collection API
     Given the following collection:
       | title | EU Interoperability Support Group |
     Then I should have 1 collection
+
+  Scenario: Assign OG ownership during creation of collections through UI
+    Given the following person:
+      | name | Person collection API example |
```

Comment:
I would like to use better names. These long composed names are hard to understand, a simple real name would reflect more its meaning.